packages/@aws-cdk-testing/framework-integ/test/aws-apigatewayv2-integrations/test/websocket/integ.aws.ts

```diff
@@ -42,6 +42,9 @@ webSocketApi.addRoute('$connect', {
     integrationUri: `arn:aws:apigateway:${stack.region}:dynamodb:action/PutItem`,
     integrationMethod: HttpMethod.POST,
     credentialsRole: apiRole,
+    requestParameters: {
```

Comment:
This won't be enough unfortunately, you'll need to build this change and run `yarn integ <path-to-this-file> --update-on-failed` to update the snapshots as well.

Reply:
Hey Gavin, I think i will need assistance on this. Unfortunately I don't know this library very well. Can you help out please?

Reply:
```
npx lerna run build --scope=@aws-cdk-testing/framework-integ
cd packages/@aws-cdk-testing/framework-integ
yarn integ test/aws-apigatewayv2-integrations/test/websocket/integ.aws.js --update-on-failed
```

Reply:
I am having some typescript compilation issues and the commands fail for me. Please feel free to add to the pr as you see fit.

Reply:
It seems that the command itself is not working. Try `yarn integ --update-on-failed` instead and it should work for you.

Reply:
running the command `yarn integ --update-on-failed` gives me yarn integ --update-on-failed Error: Cannot find module './integ-runner.js'.  The command seems to be trying to run /packages/@aws-cdk/integ-runner/bin/integ-runner but cannot find the ./integ-runner.js file and fails.